prog/dns_zone/setup_dns_server_vm.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+class Prog::DnsZone::SetupDnsServerVm < Prog::Base
+  subject_is :vm, :sshable
+
+  def self.assemble(dns_server_id, name: nil, project_id: nil, vm_size: "standard-2", storage_size_gib: 30, location: "hetzner-fsn1")
+    unless (dns_server = DnsServer[dns_server_id])
+      fail "No existing Dns Server"
+    end
+
+    project_id ||= (Config.production? || Config.dns_service_project_id) ? Config.dns_service_project_id : Project.order_by(:created_at).first.id
+    unless (project = Project[project_id])
+      fail "No existing Project"
+    end
+
+    name ||= dns_server.ubid.to_s + SecureRandom.alphanumeric(8).downcase
```

Comment:
nit: removes `.to_s`
```suggestion
    name ||= "#{dns_server.ubid}-#{SecureRandom.alphanumeric(8).downcase}"
```

Reply:
Thanks!